contracts/contracts/token/OUSD.sol

```diff
@@ -473,22 +477,19 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
             );
         }
```

Comment:
Aren't we missing a return statement here?

Reply:
The return is a few lines down

Reply:
Sorry, I don't see it. I was expecting at the end of the if statement to exit the function. I must be missing something completely obvious?

Reply:
Ohh yes you are right, good catch. The github code excerpt was showing me something weird.